### PR TITLE
Closes #153 - add logging 

### LINKED ISSUE #153: Add logging to back-end (log4net)
This task about logging user activity on server using Microsoft.Extension.Logging.Log4net

### PR DESCRIPTION
**Pull request by TarasKataryna**

### CODE REVIEW

File: gtd-timer/Controllers/LogInController.cs [hunk 1]
```diff
@@ -20,6 +21,7 @@ public LogInController(ILogInService logInService)
         [HttpPost]
         public IActionResult Login([FromBody] LoginDTO model)
         {
+
```
Comment (roman-fay) [inline gtd-timer/Controllers/LogInController.cs]: redund

File: gtd-timer/Middleware/ErrorHandlingMiddleware.cs [hunk 2]
```diff
@@ -47,6 +50,7 @@ public Task HandleExceptionAsync(HttpContext context, Exception exception)
         private HttpStatusCode GetExceptionCode(Exception exception)
         {
             HttpStatusCode exceptionCode;
+            this.logger.LogError("Error message: {0}; Error occured in: {1}, {2}", exception.Message, exception.Source,exception.TargetSite);
```
Comment (roman-fay) [inline gtd-timer/Middleware/ErrorHandlingMiddleware.cs]: can you use c#'s string interpolation here? also please format this better

File: gtd-timer/Controllers/LogInController.cs (hunk 1 above)
Comment (TarasKataryna) [reply gtd-timer/Controllers/LogInController.cs]: Deleted

File: gtd-timer/Middleware/ErrorHandlingMiddleware.cs (hunk 2 above)
Comment (TarasKataryna) [reply gtd-timer/Middleware/ErrorHandlingMiddleware.cs]: Change (in that way?)
chrome_2019-01-20_16-33-39

File: gtd-timer/Program.cs [hunk 3]
```diff
@@ -12,6 +14,7 @@ public static void Main(string[] args)
 
         public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
             WebHost.CreateDefaultBuilder(args)
-                .UseStartup<Startup>();
```
Comment (roman-fay) [inline gtd-timer/Program.cs]: previous formatting was correct. this one is not

Comment (TarasKataryna) [reply gtd-timer/Program.cs]: Returned previous

**Pull request merged**